### PR TITLE
tests: Fix warnings in tests

### PR DESCRIPTION
**Pull request by Joshua-Ashton**

### CODE REVIEW

File: tests/d3d12_crosstest.h [hunk 1]
```diff
@@ -411,6 +411,8 @@ static inline bool is_depth_clip_enable_supported(ID3D12Device *device)
 static PFN_vkGetInstanceProcAddr pfn_vkGetInstanceProcAddr;
 static bool init_vulkan_loader(void)
 {
+    void *mod;
```
Comment (DadSchoorse) [inline tests/d3d12_crosstest.h:414]: What if mod is `HMODULE`? and why does this even compile on mingw?

Comment (Joshua-Ashton) [reply tests/d3d12_crosstest.h:414]: HMODULE == void*

Comment (DadSchoorse) [reply tests/d3d12_crosstest.h:414]: you are still declaring mod two times tho in the `_WIN32` path

Comment (HansKristian-Work) [reply tests/d3d12_crosstest.h:414]: I guess it's not as easy to trigger this case since it should only hit when using vkd3d-utils based Win32 test build.

**Review by HansKristian-Work — CHANGES_REQUESTED** [review]
LGTM, just needs to fix the _WIN32 error as mentioned in discussion.

File: tests/d3d12_crosstest.h (hunk 1 above)
Comment (Joshua-Ashton) [reply tests/d3d12_crosstest.h:417]: Resolved.

**Review by HansKristian-Work — APPROVED** [review]
LGTM

**Pull request merged**